flash/graph/data.py

```diff
@@ -38,3 +62,79 @@ def load_sample(self, sample: Any, dataset: Optional[Any] = None) -> Mapping[str
         if isinstance(sample, Data):
             return sample
         return super().load_sample(sample, dataset=dataset)
+
+
+class GraphSequenceDataSource(SequenceDataSource[Data]):
```

Comment:
@ethanwharris I think I included what you mention https://github.com/PyTorchLightning/lightning-flash/pull/617/files#r677724510